### ISSUE TITLE
[Feature Request] Add copy marketid option in UI

### ISSUE THREAD

**Issue by gabbello** [AugurProject/augur#879]
**Is your feature request related to a problem? Please describe.**
there are quite a few websites already that provide services around augur platform and most of them use
market id as easiest way to find a market. From what I could find in augur UI the only way to get this id is from the URL which is not very convenient especially since there are a lot of other parameters in GET. 

**Describe the solution you'd like**

It would be nice to have easy ability to copy to clipboard marketid from pages that lsit markets and from market page/pages itself.

**Comment by adrake33** [comment]
Adding this to the list we have going for trading page tweaks to implement once the new trading page UI is finished: #728

@matt-bullock Can you please create a quick mock for where this should go & how it should look on the trading page?

**Comment by pgebheim** [comment]
@JohnDanz -- I could swear you did this?

If not, @cillianhunter0 @matt-bullock  --- should we add this to a list of improvements, or does this need to be done now?

**Comment by cillianhunter0** [comment]
It's not in the latest designs from what I can see, @matt-bullock have I missed anything?

On first glance this seems like a good idea, question though: when I copy the market ID, what am I doing with it? Ultimately we want to be able to copy and share a URL to the market, is this a stop gap until we can do that?

**Comment by pgebheim** [comment]
Good question.

I think this is kinda like how in etherscan you can copy the txn address or account address to send it around.

In the eth world people send around the addresses to random stuff alllll the time, whether to open it in their own block explorer or send it to someone or to query info from an eth node.

The actual request is probably closer to: whenever there is an address make clicking it copy.

Paul
paul@augur.net

________________________________

**Comment by gabbello** [comment]
Since i'm the requester for this feature, the need was/is to copy the marketid to use it on sites like reporters.chat, dispute.tools etc.

**Comment by pgebheim** [comment]
Ah ok that makes sense!!

Thanks for keeping an eye on the issue.

Paul
paul@augur.net

________________________________

**Comment by cillianhunter0** [comment]
Makes sense, thanks both.

**Comment by Chwy5** [comment]
its added in the trading pg redesign @pgebheim you can close if you want, there is design done and its part of feature for a separate ticket

**Comment by Chwy5** [comment]
closing
